### PR TITLE
Add recipe for repoze.lru [skip appveyor] [skip travis]

### PR DESCRIPTION
**Pull request by nicoddemus**

### CODE REVIEW

**Comment by nicoddemus** [comment]
@ocefpaf @conda-forge/help-python ready for review. 👍

File: recipes/repoze.lru/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    - setuptools
+
+test:
+  imports:
+    - repoze
+    - repoze.lru
+
+about:
+  home: http://www.repoze.org
+  license: BSD
```
Comment (ocefpaf) [inline recipes/repoze.lru/meta.yaml]: Looks like this is BSD-3-Clause "Modification_Variant"

https://fedoraproject.org/wiki/Licensing:BSD#Modification_Variant

Never saw that before :-/

Comment (nicoddemus) [reply recipes/repoze.lru/meta.yaml]: Heh me neither. What do you suggest I change this to?

Comment (ocefpaf) [reply recipes/repoze.lru/meta.yaml]: Unless I missed it I could not find it in https://spdx.org/licenses/

I would go with `BSD Modification Variant` for now.

**Comment by nicoddemus** [comment]
@ocefpaf ready 👍

**Pull request merged**